src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/Events/SampleEventThatCausesException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests.Events;
+
+[DomainEvent("sample-event-that-causes-exception", EventSchema.CloudEvent)]
+public class SampleEventThatCausesException : IDomainEvent
+{
+    public string? Message { get; set; }
+}
+
+public class FailingHandler : IDomainEventHandler<SampleEventThatCausesException>
```

Comment:
[Nit] To remove any ambiguity on what those abstractions are for:

```suggestion
public class SampleEventThatCausesExceptionDomainEvent : IDomainEvent
{
    public string? Message { get; set; }
}

public class SampleEventThatCausesExceptionDomainEventHandler : IDomainEventHandler<SampleEventThatCausesException>
```